test/tilt_commonmarkertemplate_test.rb

```diff
@@ -1,47 +1,96 @@
 require_relative 'test_helper'
 
 checked_describe 'tilt/commonmarker' do
-  it "preparing and evaluating templates on #render" do
-    template = Tilt::CommonMarkerTemplate.new { |t| "# Hello World!" }
-    assert_equal "<h1>Hello World!</h1>\n", template.render
-  end
+  if ENV['LATEST_COMMONMARKER'] == 'yes'
```

Comment:
This should be switched to `if defined?(::Commonmarker)` if you want to test stuff that only works on commonmarker >= 1.  However, in general we would want the same specs to pass in both pre-1.0 and post-1.0 versions of commonmarker.

Adding support for new keys is fine, but can we also map old option keys to new option keys if the new version is used, so the same tilt code can work regardless of the underlying commonmarker version, assuming the new commonmarker supports the same feature as the previous version using a different option name?